iam-login-service/src/main/webapp/resources/iam/apps/dashboard-app/components/clients/client/scopelist/scopelist.component.html

```diff
@@ -45,7 +45,8 @@
                 </div>
             </div>
         </div>
-        <div class="form-group">
+        <div class="form-group"
+            ng-show="($root.loggedUser.isAdmin && $root.isAdminOnlyCustomScope) || (!$root.isAdminOnlyCustomScope)">
```

Comment:
I think it should be moved before `<label>Custom scopes</label>` to also hide **Custom scopes** from normal users.